code/modules/mob/living/simple_animal/abnormality/he/drifting_fox.dm

```diff
@@ -0,0 +1,158 @@
+// this was a mistake.
+// By yours truely, Mori.
+#define STATUS_EFFECT_FALSEKIND /datum/status_effect/false_kindness
+/mob/living/simple_animal/hostile/abnormality/drifting_fox
+	name = "Drifting Fox"
+	desc = "A large shaggy fox with gleaming yellow eyes; And torn umbrellas lodged into its back."
+	icon = 'ModularTegustation/Teguicons/96x96.dmi'
+	icon_state = "drifting_fox"
+	icon_living = "drifting_fox"
+	icon_dead = "fox_egg"
+	deathmessage = "Collapses into a glass egg"
+	deathsound = 'sound/abnormalities/drifting_fox/fox_death_sound.ogg'
+	pixel_x = -24
+	pixel_y = -26
+	base_pixel_x = -24
+	base_pixel_y = -26
+	del_on_death = FALSE
+	maxHealth = 1000
+	health = 1000
+	rapid_melee = 3
+	move_to_delay = 2
+	damage_coeff = list( RED_DAMAGE = 0.9, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 1.5 )
+	melee_damage_lower = 5
+	melee_damage_upper = 35 // Idea taken from the old PR, have a large damage range to immitate its fucked rolls and crit chance.
+	melee_damage_type = BLACK_DAMAGE
+	stat_attack = HARD_CRIT
+	attack_sound = 'sound/abnormalities/drifting_fox/fox_melee_sound.ogg'
+	attack_verb_simple = "thwacks"
+	attack_verb_continuous = "thwacks"
+	can_breach = TRUE
+	threat_level = HE_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 45,
+		ABNORMALITY_WORK_INSIGHT = 45,
+		ABNORMALITY_WORK_ATTACHMENT = list(25,30,35,40,45),
+		ABNORMALITY_WORK_REPRESSION	= 0,
+	)
```

Comment:
```suggestion
	work_chances = list(
		ABNORMALITY_WORK_INSTINCT = 40,
		ABNORMALITY_WORK_INSIGHT = 40,
		ABNORMALITY_WORK_ATTACHMENT = list(15, 20, 25, 30, 35),
		ABNORMALITY_WORK_REPRESSION	= 0,
	)
```
Makes it train attachment via petting and keeps it with decent rates on the others. Rates shouldn't be super high seeing as it barely cares about anything that isn't a failure.